.nodemonrc

```diff
@@ -1,3 +1,3 @@
 {
-  "watch": ["lib", ".env"]
+  "watch": ["."]
```

Comment:
I'm unfamiliar with performance impacts of file watches.  At least, would hope we also have meaningful ignores.  Otherwise if goal is to expand file watches to outside files this is understandable.

Reply:
Previously `nodemon` was only watching files in `lib` and the `.env` file. This change causes `nodemon` to watch for changes on any files in the current directory. The performance impact during local dev didn't appear to be noticable, or at least it did restart the process when I edited `./index.js`